server/ai_mediaserver.go

```diff
@@ -88,11 +89,20 @@ func startAIMediaServer(ctx context.Context, ls *LivepeerServer) error {
 	ls.HTTPMux.Handle("/text-to-speech", oapiReqValidator(aiMediaServerHandle(ls, jsonDecoder[worker.GenTextToSpeechJSONRequestBody], processTextToSpeech)))
 
 	// This is called by the media server when the stream is ready
-	ls.HTTPMux.Handle("/live/video-to-video/{stream}/start", ls.StartLiveVideo())
-	ls.HTTPMux.Handle("/live/video-to-video/{prefix}/{stream}/start", ls.StartLiveVideo())
-	ls.HTTPMux.Handle("/live/video-to-video/{stream}/update", ls.UpdateLiveVideo())
+	ls.HTTPMux.Handle("POST /live/video-to-video/{stream}/start", ls.StartLiveVideo())
+	ls.HTTPMux.Handle("POST /live/video-to-video/{prefix}/{stream}/start", ls.StartLiveVideo())
+	ls.HTTPMux.Handle("POST /live/video-to-video/{stream}/update", ls.UpdateLiveVideo())
 	ls.HTTPMux.Handle("/live/video-to-video/smoketest", ls.SmokeTestLiveVideo())
 
+	// Configure WHIP ingest only if an addr is specified.
+	// TODO use a proper cli flag
+	if os.Getenv("LIVE_AI_WHIP_ADDR") != "" {
```

Comment:
How about moving this to flags? I think we don't use anywhere the env variables directly in go-livepeer, but just adding more flags.

Reply:
Yes it has been mentioned in several places this can be done as a follow up to this PR